lib/sqlsync/src/lsn.rs

```diff
@@ -5,6 +5,7 @@ use std::{
 
 use serde::{Deserialize, Serialize};
 
+/// Log Sequence Number
```

Comment:
I think this is the only actual change in this file? Can you revert the other changes since I think they are just due to our rustfmt settings differing. I'll figure out how to enforce consistent rustfmt settings for the whole repo to prevent this in the future.